README.md

```diff
@@ -759,10 +767,17 @@ and `borders` (for `min` and `max` calls).
 Each operation section consists of different collectors
 for success calls and error (both error throw and `nil, err`)
 returns. `count` is the total requests count since instance start
-or stats restart. `latency` is the 0.99 quantile of request execution
-time if `metrics` driver used and quantiles enabled,
-otherwise `latency` is the total average.
-`time` is the total time of requests execution.
+or stats restart.  `time` is the total time of requests execution.
+`latency_average` is `time` / `count`.
+`latency_quantile_recent` is the 0.99 quantile of request execution
+time for a recent period (see 
+[`metrics` summary API](https://www.tarantool.io/ru/doc/latest/book/monitoring/api_reference/#summary)).
+It is computed only if `metrics` driver used and quantiles enabled.
+`latency_quantile_recent` value may be `-nan` if there wasn't any
+observations for several ages, see
+[tarantool/metrics#303](https://github.com/tarantool/metrics/issues/303).
+`latency` is a `latency_quantile_recent`, if it presents,
+otherwise it's `latency_average`.
```

Comment:
I can understand it as composition of the following three conditions:

1. Metrics is enabled.
2. Quantiles are enabled.
3. `latency_quantile_recent` is `-nan` at the moment (it means that no data is present, right?)

However, as I remember from our private discussion, you had idea to implement it based on the first two conditions.

It requires a clarification in the documentation to avoid misinterpretation.

Reply:
It's only the first two, yes. If `latency_quantile_recent` is `-nan`, `latency` is `-nan` too.

Reply:
Replaced "if it presents" with "if the field in not empty". I'm not sure we should explicitly write about `-nan` here.

Reply:
But why not 'if metrics and quantiles are enabled'?

Reply:
I just thought it will be tautologic. But your version is better, so I reworked it.

Reply:
I think that the latest variant is more explicit. Thanks for the update.